frontend/app/component/form/data_entry/candidates_votes/CandidatesVotesForm.tsx

```diff
@@ -139,11 +57,20 @@ export function CandidatesVotesForm({ group }: CandidatesVotesFormProps) {
       id={`candidates_form_${group.number}`}
       title={`${t("list")} ${group.number} - ${group.name}`}
     >
-      {isSaved && hasValidationError && (
-        <Feedback id="feedback-error" type="error" data={errors.map((error) => error.code)} />
+      {error instanceof ApiError && <ErrorModal error={error} />}
+      <DataEntryNavigation
+        onSubmit={onSubmit}
+        currentValues={{
+          political_group_votes: pollingStationResults.political_group_votes.map((pg) =>
+            pg.number !== group.number ? pg : formValuesToValues(currentValues),
+          ),
+        }}
+      />
+      {formSection.isSaved && formSection.errors.length > 0 && (
+        <Feedback id="feedback-error" type="error" data={formSection.errors.map((error) => error.code)} />
       )}
-      {isSaved && hasValidationWarning && !hasValidationError && (
-        <Feedback id="feedback-warning" type="warning" data={warnings.map((warning) => warning.code)} />
+      {formSection.isSaved && formSection.warnings.length > 0 && formSection.errors.length === 0 && (
```

Comment:
```suggestion
      {formSection.isSaved && showAcceptWarnings && (
```
my suggestion would be to rename `showAcceptWarnings` to `hasWarnings`
Same suggestion for `DifferencesForm.tsx`

Reply:
this is a bool to indicate if the accept warnings checkbox should be shown. which is if there are warnings and no errors. 
Do agree this is getting complicated.

Reply:
it's in the use hook concisely now.